docs/Welcome.mdx

```diff
@@ -142,24 +142,35 @@ UI5 Web Components for React supports [TypeScript](https://www.typescriptlang.or
 
 You can find all available interfaces of the main package [here](https://github.com/SAP/ui5-webcomponents-react/tree/main/packages/main/src/interfaces) and for charts [here](https://github.com/SAP/ui5-webcomponents-react/tree/main/packages/charts/src/interfaces).
 
-<br />
+#### Example
 
-Example - `Popover` with `Ref` type declaration:
+Small app with a popover opened by clicking a button including type declarations:
 
 ```tsx
-import React, { MouseEvent, useRef } from 'react';
-import { ThemeProvider, Button, Popover, PopoverDomRef } from '@ui5/webcomponents-react';
+import React, { useState, useRef } from 'react';
+import type { ButtonPropTypes, PopoverDomRef, PopoverPropTypes } from '@ui5/webcomponents-react';
+import { ThemeProvider, Button, Popover } from '@ui5/webcomponents-react';
 
 export default function App() {
+  const [open, setOpen] = useState<PopoverPropTypes['open']>(false);
   const popoverRef = useRef<PopoverDomRef>(null);
-  const onClick = (e: MouseEvent<HTMLElement, MouseEvent>) => {
-    popoverRef.current?.showAt(e.target);
+
+  const handleClick: ButtonPropTypes['onClick'] = (e) => {
+    setOpen(true);
+  };
+  // if you only want to type the event parameter, you can use the `Parameters` utility type.
+  const handleAfterClose = (e: Parameters<PopoverPropTypes['onAfterClose']>[0]) => {
```

Comment:
```suggestion
  const handleAfterClose: PopoverPropTypes['onAfterClose'] = (e) => {
```

Reply:
I would not include the Parameters hint here, this should not be necessary for a regular user of UI5 Web Components, so I think having this example directly in on our Welcome Page is rather confusing